src/libexec/zmfixperms

```diff
@@ -706,6 +706,16 @@ if [ -x /opt/zimbra/common/sbin/nginx ]; then
   chown ${root_user}:${zimbra_group} /opt/zimbra/common/sbin/nginx
   chmod 750 /opt/zimbra/common/sbin/nginx
 
+  if [ -x /opt/zimbra/log/nginx.log ]; then
+    chown ${zimbra_user}:${zimbra_group} /opt/zimbra/log/nginx.log
+    chmod 644 /opt/zimbra/log/nginx.log
+  fi
+
+  if [ -x /opt/zimbra/log/nginx.access.log ]; then
+    chown ${zimbra_user}:${zimbra_group} /opt/zimbra/log/nginx.access.log
+    chmod 644 /opt/zimbra/log/nginx.access.log
+  fi
```

Comment:
Are you sure you want `-x` here for your tests and not `-f` or `-e`? Do you actually mean to test if the file is executable, or only that it exists?

Reply:
Good find, it was a copy paste mistake :)